src/jats/jats-exporter.ts

```diff
@@ -1749,57 +1752,25 @@ export class JATSExporter {
           }
         })
       }
-      const noteIDs: string[] = []
-      for (const contributor of [...authorContributors, ...otherContributors]) {
-        if (contributor.footnote) {
-          const ids = contributor.footnote.map((note) => {
-            return note.noteID
-          })
-          noteIDs.push(...ids)
-        }
-        if (contributor.corresp) {
-          const ids = contributor.corresp.map((corresp) => {
-            return corresp.correspID
-          })
-          noteIDs.push(...ids)
-        }
-      }
-      const footnotes: Footnote[] = []
-      footnotes.push(
-        ...this.models.filter(hasObjectType<Footnote>(ObjectTypes.Footnote))
-      )
-
-      const correspodings: Corresponding[] = []
-      correspodings.push(
-        ...this.models.filter(
-          hasObjectType<Corresponding>(ObjectTypes.Corresponding)
-        )
-      )
-
-      if (footnotes || correspodings) {
+      const authorNotes = getModelsByType<AuthorNotes>(
+        this.modelMap,
+        ObjectTypes.AuthorNotes
+      )[0]
```

Comment:
Please change this to `)?.[0]` otherwise it will break when no author notes exist.

Reply:
Oh ok I didn't know that.